src/game_map.cpp

```diff
@@ -1262,8 +1262,25 @@ bool Game_Map::PrepareEncounter(BattleArgs& args) {
 
 	args.troop_id = encounters[Rand::GetRandomNumber(0, encounters.size() - 1)];
 
-	if (Rand::GetRandomNumber(1, 32) == 1) {
-		args.first_strike = true;
+	if (Player::IsRPG2k()) {
+		if (Rand::GetRandomNumber(1, 32) == 1) {
```

Comment:
https://github.com/EasyRPG/Player/blob/ff36dfd32318fae14300ed562f5690ab94dfaf1c/src/rand.cpp#L81-L84
could be used here.

Reply:
Did the requested change now.